widgets/toggle.js

```diff
@@ -104,7 +104,14 @@ function prepareToggleData(transformData) {
       count: data.count
     };
 
-    return transformData && transformData(newData) || newData;
+    if (transformData) {
+      newData = transformData(newData);
+      if (typeof newData !== 'object') {
+        throw new Error('`transformData` must return an object.');
+      }
+    }
```

Comment:
That's me that did this because I needed to have a default transform data since the exposed. You should just be able to call transform data and let the template do the if throw.